src/types/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum Error {
```

Comment:
Consider renaming the enum to 'ErrorMessage' for better readability and understanding.